src/api/Feed.js

```diff
@@ -943,6 +949,70 @@ class Feed extends Base {
         }
     }
 
+    /**
+     * Fetches app activities for a file
+     *
+     * @return {Promise} - the feed items
+     */
+    fetchAppActivity(): Promise<?AppActivityItems> {
+        this.appActivityAPI = new AppActivityAPI(this.options);
+
+        return new Promise(resolve => {
+            this.appActivityAPI.getAppActivity(this.id, resolve, this.fetchFeedItemErrorCallback.bind(this, resolve));
+        });
+    }
+
+    /**
+     * Deletes an app activity item.
+     *
+     * @param {BoxItem} file - The file to which the app activity belongs to
+     * @param {AppActivityItem} appActivityItem - The app activity item to delete
+     * @param {BoxItemPermission} permissions - Permissions for the app activity
+     * @param {Function} successCallback - the function which will be called on success
+     * @param {Function} errorCallback - the function which will be called on error
+     * @return {void}
+     */
+    deleteAppActivity = (
+        file: BoxItem,
+        appActivityItem: AppActivityItem,
+        permissions: BoxItemPermission,
+        successCallback: Function,
+        errorCallback: ErrorCallback,
+    ): void => {
+        this.appActivityAPI = new AppActivityAPI(this.options);
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        const { id: activityItemId } = appActivityItem.id;
```

Comment:
Should this just be `const { id: activityItemId } = appActivityItem;`

Reply:
Good catch, thanks